### PR TITLE
Add health api document.

### PR DESCRIPTION
**Pull request by vwxyzh**

### CODE REVIEW

File: docs/howto-tsg.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
         - [Service-side logs](#add_logs_service)
 - [Serverless mode troubleshooting](#serverless_mode_tsg)
 - [Classic mode troubleshooting](#classic_mode_tsg)
+- [Service health](#service_health)
```
Comment (JialinXin) [inline docs/howto-tsg.md]: `#service_health` link not work. Need add `<a>` link before the header in that section, or directly use header way like `#Service-health`

File: docs/howto-tsg.md [hunk 2]
```diff
@@ -143,4 +144,18 @@ To diagnose connectivity issues in `Serverless` mode, the most straight forward
...
 
-Troubleshooting `classic` mode connectivity and message delivary issues are similar to [troubleshooting default mode issues](#default_mode_tsg).
\ No newline at end of file
+Troubleshooting `classic` mode connectivity and message delivary issues are similar to [troubleshooting default mode issues](#default_mode_tsg).
+
+## Service health
+
+You can check the health api for service health.
+
+* Address: `https://{instance_name}.service.signalr.net/v1/api/health`
```
Comment (JialinXin) [inline docs/howto-tsg.md]: Request and Response ?

File: docs/howto-tsg.md [hunk 3]
```diff
@@ -143,4 +144,18 @@ To diagnose connectivity issues in `Serverless` mode, the most straight forward
...
+* Status code:
+  * 200: healthy.
+  * 503: your service is unhealthy.
+    You can:
+    * Wait for auto recover.
+    * Or restart instance.
+    
+    If either not work, please concract us.
```
Comment (JialinXin) [inline docs/howto-tsg.md]: typo: contact?

Comment (JialinXin) [reply docs/howto-tsg.md]: both not work?

File: docs/howto-tsg.md [hunk 4]
```diff
@@ -143,4 +144,19 @@ To diagnose connectivity issues in `Serverless` mode, the most straight forward
...
+
+* Request url: `https://{instance_name}.service.signalr.net/v1/api/health`
+* Response status code:
+  * 200: healthy.
+  * 503: your service is unhealthy.
+    You can:
+    * Wait for auto recover.
+    * Or restart instance.
+    
+    If both not work, please concract us.
```
Comment (JialinXin) [inline docs/howto-tsg.md]: typo: contact?

File: docs/howto-tsg.md [hunk 5]
```diff
@@ -143,4 +144,19 @@ To diagnose connectivity issues in `Serverless` mode, the most straight forward
...
+## Service health
+
+You can check the health api for service health.
+
+* Request url: `https://{instance_name}.service.signalr.net/v1/api/health`
+* Response status code:
+  * 200: healthy.
+  * 503: your service is unhealthy.
+    You can:
+    * Wait for auto recover.
```
Comment (vicancy) [inline docs/howto-tsg.md]: maybe not `auto-recover` ...

File: docs/howto-tsg.md [hunk 6]
```diff
@@ -143,4 +144,19 @@ To diagnose connectivity issues in `Serverless` mode, the most straight forward
...
+    * Or restart instance.
+    
+    If both not work, please contact us.
```
Comment (vicancy) [inline docs/howto-tsg.md]: shall we also provide a link to the disaster recovery solution? https://docs.microsoft.com/en-us/azure/azure-signalr/signalr-concept-disaster-recovery

File: docs/howto-tsg.md [hunk 7]
```diff
@@ -143,4 +144,19 @@ To diagnose connectivity issues in `Serverless` mode, the most straight forward
...
+
+<a name="service_health"></a>
+## Service health
+
+You can check the health api for service health.
+
+* Request url: `https://{instance_name}.service.signalr.net/v1/api/health`
+* Response status code:
+  * 200: healthy.
+  * 503: your service is unhealthy.
```
[the remark this answers is not in the document]
Comment (vwxyzh) [reply docs/howto-tsg.md]: its hard to give reasons without the implements detail...

**Pull request merged**